XpringKit/JavaScript/XRPJavaScriptLoader.swift

```diff
@@ -61,7 +61,7 @@ internal enum XRPJavaScriptLoader {
     // TODO(keefer): Drop context parameter when context becomes a singleton.
     public static func load(_ resourceName: String, from context: JSContext) -> JSValue {
         guard
-            let entrypoint = context.objectForKeyedSubscript("EntryPoint"),
+            let entrypoint = context.objectForKeyedSubscript("XpringCommonJS"),
```

Comment:
Please update comment on line 53.

Reply:
Should we call this variable something besides entrypoint? `xpringCommon`, `xpringCommonJS` or `jsRoot`? I'm not sure what the right terminology would be.